Indicators/RelativeVolume.cs

```diff
@@ -0,0 +1,81 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using QuantConnect.Data.Market;
+
+namespace QuantConnect.Indicators
+{
+    /// <summary>
+    /// Represents the relative volume indicator (RVOL)
+    /// </summary>
+    public class RelativeVolume : WindowIndicator<TradeBar>, IIndicatorWarmUpPeriodProvider
+    {
+        /// <summary>
+        /// A rolling sum for computing the average for the given period
+        /// </summary>
+        public IndicatorBase<IndicatorDataPoint> RollingSum { get; }
+
+        /// <summary>
+        /// Gets a flag indicating when this indicator is ready and fully initialized
+        /// </summary>
+        public override bool IsReady => RollingSum.IsReady;
+
+        /// <summary>
+        /// Resets this indicator to its initial state
+        /// </summary>
+        public override void Reset()
+        {
+            RollingSum.Reset();
+            base.Reset();
+        }
```

Comment:
Usually in Lean constructors are placed at the top, after attributes and properties, and before any other methods.